XmlValidator/spec_exercise.xml

```diff
@@ -278,10 +278,11 @@
   <entry>
     <xpath>//entry/response/query</xpath>
     <children>
-      <one-of>
+      <any-number>
        <reference>block-or-minor-environment-elements</reference>
        <reference>inline-elements</reference>
-      </one-of>
+       <element>radio</element>
+      </any-number>
```

Comment:
I manged to figure out where to add the radio block element. So there is no problem now.